### PR TITLE
Signed channels

### PR DESCRIPTION
**Pull request by ddkohler**

### CODE REVIEW

**Comment by ksunden** [comment]
I would recommend implementing the external message as a dict, (avro map) rather than a list

This is what we do for channel shapes, for instance

Also the fallback should should probably be computed in the message rather than in `__init__`

**Comment by ksunden** [comment]
internal state can probably remain a list, I guess, but I may make it a dict just for conceptual consistency.

File: yaqd_ni/ni-daqmx-tmux.toml [hunk 1]
```diff
@@ -95,7 +95,12 @@ default = 0
 [messages]
 
 [messages.get_channel_signs]
-response = [{"type"="array", "items"="boolean"}]
+doc = """Returns a map of each channel to a boolean.  \\
```
Comment (ksunden) [inline yaqd_ni/ni-daqmx-tmux.toml:98]: Not sure why you added `\\` here...

**Comment by ddkohler** [comment]
I think it has been determined afk that the proper fix is upstream (augment the `is-sensor` trait).  This daemon will wait for new trait features.

**Pull request closed without merging**